webapp/controller/ListSelector.js

```diff
@@ -4,14 +4,14 @@ sap.ui.define([
 	], function (BaseObject, GroupHeaderListItem) {
 		"use strict";
 
-		return BaseObject.extend("sap.ui.demo.masterdetail.model.ListSelector", {
+		return BaseObject.extend("sap.ui.demo.masterdetail.demoapp.controller.ListSelector", {
```

Comment:
please remove the "demoapp" part in the namespace
We need to change this template-related coding again so that the namespace is: sap.ui.demo
And the app id is: masterdetail